src/styles/components/breadcrumbs.less

```diff
@@ -142,3 +142,11 @@ Breadcrumbs
     }
   }
 }
+
+.breadcrumb-style-headline {
+  margin-bottom: @base-spacing-unit * 2/3;
+  .h4 {
+    font-size: @base-spacing-unit * .65;
```

Comment:
Good shout! I'm going to resist the `line-height` suggestion because (possibly wrongly) the breadcrumb uses `base-spacing-unit` and I'd rather keep the two in line than deviate. Also, 24/13 is not a nice number...